airflow/providers/databricks/sensors/sql.py

```diff
@@ -0,0 +1,135 @@
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+"""This module contains Databricks sensors."""
+
+from __future__ import annotations
+
+from typing import Any, Callable, Sequence
+
+from airflow.providers.common.sql.hooks.sql import fetch_all_handler
+from airflow.providers.databricks.hooks.databricks_sql import DatabricksSqlHook
+from airflow.sensors.base import BaseSensorOperator
+from airflow.utils.context import Context
+
+
+class DatabricksSqlSensor(BaseSensorOperator):
+    """Generic SQL sensor for Databricks
+
+    :param databricks_conn_id: connection id from Airflow to databricks,
+        defaults to DatabricksSqlHook.default_conn_name
+    :param http_path: Optional string specifying HTTP path of
+        Databricks SQL Endpoint or cluster.If not specified, it should be either specified
+        in the Databricks connection's extra parameters, or ``sql_endpoint_name`` must be specified.
+    :param sql_endpoint_name: Optional name of Databricks SQL Endpoint.
+        If not specified, ``http_path`` must be provided as described above.
+    :param session_configuration: An optional dictionary of Spark session parameters.
+        Defaults to None. If not specified, it could be specified in the
+        Databricks connection's extra parameters.
+    :param http_headers: An optional list of (k, v) pairs that will be set as HTTP headers on every request.
+    :param client_parameters: Additional parameters internal to Databricks SQL Connector parameters.
+    :param sql: SQL query to be executed.
+    :param catalog: Database name/catalog name, defaults to "hive_metastore"
+    :param schema: Schema name, defaults to "default"
+    :param handler: The result handler which is called with the result of each statement.
+    """
+
+    template_fields: Sequence[str] = (
+        "databricks_conn_id",
+        "sql",
+        "catalog",
+        "schema",
+        "http_headers",
+    )
+
+    template_ext: Sequence[str] = (".sql",)
+    template_fields_renderers = {"sql": "sql"}
+
+    def __init__(
+        self,
+        *,
+        databricks_conn_id: str = DatabricksSqlHook.default_conn_name,
+        http_path: str | None = None,
+        sql_endpoint_name: str | None = None,
+        session_configuration=None,
+        http_headers: list[tuple[str, str]] | None = None,
+        catalog: str = "hive_metastore",
+        schema: str = "default",
+        sql: str | None = None,
+        handler: Callable[[Any], Any] = fetch_all_handler,
+        client_parameters: dict[str, Any] | None = None,
+        **kwargs,
+    ) -> None:
+        """_summary_
+
+        :param databricks_conn_id: _description_, defaults to DatabricksSqlHook.default_conn_name
```

Comment:
What is the `_description_` that appears on every parameter?

Reply:
Sorry, missed the docstring for the file. Now fixed.